qcodes/tests/parameter/test_parameter_basics.py

```diff
@@ -1,7 +1,7 @@
 import pytest
 
 import qcodes.utils.validators as vals
-from qcodes.instrument.function import Function
+from qcodes.instrument import Function
```

Comment:
It's starting to smell like Functions should be in parameters module ? (not for this pr thou)

Reply:
Done as part of https://github.com/QCoDeS/Qcodes/pull/4286